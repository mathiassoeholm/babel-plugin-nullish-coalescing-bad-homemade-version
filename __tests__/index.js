import pluginTester from "babel-plugin-tester";

import plugin from "../src";

pluginTester({
  plugin,
  snapshot: true,
  tests: [
    {
      title: "transform in function",
      code: `
        function foo(opts) {
          var foo = opts.foo ?? "default";
        }
      `,
    },
  ],
});
