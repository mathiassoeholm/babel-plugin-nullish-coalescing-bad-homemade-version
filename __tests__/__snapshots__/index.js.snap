// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`unknown plugin transform in function: transform in function 1`] = `

function foo(opts) {
  var foo = opts.foo ?? "default";
}

      ↓ ↓ ↓ ↓ ↓ ↓

function foo(opts) {
  var _opts$foo = opts.foo;
  var foo = _opts$foo !== null && _opts$foo !== void 0 ? _opts$foo : "default";
}


`;
